docs/application/dotnet/samples/ui/overview.md

```diff
@@ -307,57 +242,7 @@ for TD:
 			</td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/NUI/FirstScreen" target="_blank"><strong>(T) FirstScreen (NUI)</strong></a></p>
-            <p>This sample application demonstrates how to launch the apps on the TV.</p>
-			</td>
-		</tr>
-		<tr>
-			<td>
-			<p><img alt="" height="225" src="media/tv7flexcontainersample.png" width="400" /></p>
-			</td>
-			<td>
-			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/NUI/FlexContainerSample" target="_blank"><strong>(T) FlexContainer Sample (NUI)</strong></a></p>
```

Comment:
Was it verified if these samples are referenced in any other files, I did find some of them referred in other guides however they seem to orphan guides not appearing in the TOC.

Example I could see that FlexContainer Sample is referred in https://github.com/Samsung/tizen-docs/commits/master/docs/application/dotnet/guides/nui/flexcontainer.md however I do not find flexcontainer.md topic in published documents.

Reply:
I heard some news from NUI team one month ago. They have a plan revising NUI document. It might be already done or under construction... hm.. anyway, I will ask them to remove invalid sample link from NUI guide.

Reply:
@sgchoi5 I am merging this PR, but please ensure none of the removed samples are referred elsewhere.